src/fundamental/macro-fundamental.h

```diff
@@ -249,6 +249,30 @@
                         CONST_ISPOWEROF2(_x);                          \
                 }))
 
+#define ADD_SAFE(ret, a, b) (!__builtin_add_overflow((a), (b), (ret)))
+#define INC_SAFE(a, b) __INC_SAFE(UNIQ, (a), (b))
```

Comment:
Please remove unneeded parens (534fc25ad9d83069bbe63ea2eecc6d6d5ce43528)

Reply:
Done. Please double-check that I did it right